Makefile

```diff
@@ -5,6 +5,7 @@ GOBUILD=go build
 GOPATH=$(TARGET_DIR):$(TARGET_DIR)/vendor:$(CURPATH)/cmd
 
 DOCKER_OPTS=
+IMAGE_BUILD=docker build
```

Comment:
shouldn't this be in quotes? `IMAGE_BUILD="docker build"`

Reply:
i don't 'think' it matters.  my version of docker doesnt support multistage so i had to test using image builder... but the 'docker build' command otherwise failed...

Reply:
@ewolinetz per [1] not required

[1] https://ftp.gnu.org/old-gnu/Manuals/make-3.79.1/html_chapter/make_6.html#SEC64